src/app/shared/components/spinner/spinner.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="spinner-container">
```

Comment:
Avoid "wrapper" elements, and root elements in aomponent teplate in general. Component itself is an element. you can add classes to the component element itself with
```ts
@Component({
    ...
    host: {
        class: 'my-class another-class'
    }
})
```